sftp/1.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,99 @@
+= Troubleshoot SFTP Connector
+
+To troubleshoot Anypoint Connector for SFTP (SFTP Connector), review the following sections for performing general troubleshooting, addressing specific common errors, and interpreting commonly thrown messages.
+
+== Troubleshoot Connectivity, Permissions and Server Side Issues
+
+Sometimes when attempting to connect to an SFTP server you can run into issues to establish the connection because an SFTP requires a configured encrypted communication channel.
```

Comment:
```suggestion
Sometimes when attempting to connect to an SFTP server you can experience problems establishing the connection because an SFTP requires a configured encrypted communication channel.
```